recipes/wilzegers_autotest/all/test_package/test_package.cpp

```diff
@@ -0,0 +1 @@
+#include <autotest/autotest.hpp>
#include <fstream>

int main() {
    AutoTest::Args::integralRange(1, 3);
    return 0;
}

```

Comment:
This garbling due to mixed encoding is new to me..
```suggestion
#include <autotest/autotest.hpp>
#include <fstream>

int main() {
    AutoTest::Args::integralRange(1, 3);
    return 0;
}

```

Reply:
Fix. It was ```LF``` instead of ```CR```